### PR TITLE
Add unit support

### PR DESCRIPTION
**Pull request by juliohm**

### CODE REVIEW

File: src/extra/stl.jl [hunk 1]
```diff
@@ -56,18 +53,16 @@ function stlasciiread(fname)
...
-  vertices = NTuple{3,Point3f}[]
+  normals = NTuple{3,Float32}[]
+  vertices = NTuple{3,NTuple{3,Float32}}[]
 
   open(fname) do io
     skip(io, 80) # skip header
     ntriangles = read(io, UInt32)
     for _ in 1:ntriangles
-      normal = Vec(ntuple(i -> read(io, Float32), 3))
+      normal = ntuple(_ -> read(io, Float32), 3)
```
Comment (eliascarv) [inline src/extra/stl.jl:63]: An alternative syntax for this would be:
```julia
normal = ntuple(Returns(read(io, Float32)), 3)
```

File: test/runtests.jl [hunk 2]
```diff
@@ -137,18 +139,18 @@ end
 
     @testset "STL" begin
       gtb = GeoIO.load(joinpath(datadir, "tetrahedron_ascii.stl"))
-      @test eltype(gtb.NORMAL) <: Vec3
+      @test eltype(gtb.NORMAL) <: Vec{3,Meshes.Met{Float64}}
```
Comment (juliohm) [inline test/runtests.jl:142]:
```suggestion
      @test eltype(gtb.NORMAL) <: Vec{3}
```

File: test/runtests.jl [hunk 3]
```diff
@@ -137,18 +139,18 @@ end
...
       @test gtb.geometry isa SimpleMesh
       @test embeddim(gtb.geometry) == 3
-      @test coordtype(gtb.geometry) <: Float64
+      @test Meshes.lentype(gtb.geometry) <: Meshes.Met{Float64}
       @test eltype(gtb.geometry) <: Triangle
       @test length(gtb.geometry) == 4
 
       gtb = GeoIO.load(joinpath(datadir, "tetrahedron_bin.stl"))
-      @test eltype(gtb.NORMAL) <: Vec3f
+      @test eltype(gtb.NORMAL) <: Vec{3,Meshes.Met{Float32}}
```
Comment (juliohm) [inline test/runtests.jl:150]:
```suggestion
      @test eltype(gtb.NORMAL) <: Vec{3}
```

File: test/data/tetrahedron_ascii.stl [hunk 4]
```diff
@@ -1,26 +1,26 @@
...
-facet normal 0.000000e+00 -5.000000e-01 0.000000e+00
+facet normal 0.000000e+00 -1.000000e+00 0.000000e+00
     outer loop
         vertex 0.000000e+00 0.000000e+00 0.000000e+00
         vertex 1.000000e+00 0.000000e+00 0.000000e+00
         vertex 0.000000e+00 0.000000e+00 1.000000e+00
     endloop
 endfacet
-facet normal 5.000000e-01 5.000000e-01 5.000000e-01
+facet normal 5.773503e-01 5.773503e-01 5.773503e-01
```
Comment (juliohm) [inline test/data/tetrahedron_ascii.stl:16]: is this change related to units?
